bin/sh/install-boost.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 if [ ! -d "$BOOST_ROOT/lib" ]
 then
-  wget $BOOST_URL -O /tmp/boost.tar.gz
+  wget --no-check-certificate $BOOST_URL -O /tmp/boost.tar.gz
```

Comment:
One other option if this is not appealing is to try cloning the boost project directly from git and checking out the right tag.